composer.json

```diff
@@ -10,12 +10,12 @@
         }
     ],
     "require": {
-        "php": ">=5.4.0",
-        "illuminate/support": "4.*|5.*"
+        "php": ">=7.2.0",
```

Comment:
Can this support 7.0 or 7.1? I don’t mind dropping 5.x but a lot of folks are still on older minor revs of 7

Reply:
Also can you update the TravisCI config to drop 5.x in the test plan?